core/src/migrations/000054-lockedString.ts

```diff
@@ -23,8 +23,8 @@ export default {
       const table = tables[i];
       await queryInterface.changeColumn(table, "locked", {
         type: DataTypes.STRING(191),
+        defaultValue: null, // normally the default value of a string column is null, but since we are migrating from a boolean, we need to re-set the default
```

Comment:
Comment added because this adding a `defaultValue=null` is not the normal pattern